hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/ObjectStore.java

```diff
@@ -731,19 +732,75 @@ public CancelSnapshotDiffResponse cancelSnapshotDiff(String volumeName,
 
   /**
    * Get a list of the SnapshotDiff jobs for a bucket based on the JobStatus.
-   * @param volumeName Name of the volume to which the snapshotted bucket belong
-   * @param bucketName Name of the bucket to which the snapshots belong
-   * @param jobStatus JobStatus to be used to filter the snapshot diff jobs
-   * @param listAll Option to specify whether to list all jobs or not
-   * @return a list of SnapshotDiffJob objects
+   *
+   * @param volumeName          Name of the volume to which the snapshotted bucket belong
+   * @param bucketName          Name of the bucket to which the snapshots belong
+   * @param jobStatus           JobStatus to be used to filter the snapshot diff jobs
+   * @param listAll             Option to specify whether to list all jobs or not
+   * @param prevSnapshotDiffJob list snapshot diff jobs after this snapshot diff job.
+   * @return an iterator of SnapshotDiffJob objects
    * @throws IOException in case there is a failure while getting a response.
    */
-  public List<OzoneSnapshotDiff> listSnapshotDiffJobs(String volumeName,
-                                                    String bucketName,
-                                                    String jobStatus,
-                                                    boolean listAll)
+  public Iterator<OzoneSnapshotDiff> listSnapshotDiffJobs(String volumeName,
+                                                          String bucketName,
+                                                          String jobStatus,
+                                                          boolean listAll,
+                                                          String prevSnapshotDiffJob)
```

Comment:
This diff illustrates why we shouldn't indent method signature like this: have to reindent all lines when only return type is changed.

Please avoid adding new methods formatted like this as well as changing existing methods to be like this.

Reply:
Changed the signature to avoid it in the future.